.gitignore

```diff
@@ -1,2 +1,3 @@
+/Cargo.lock
```

Comment:
As far as I know, the Cargo.lock should be there for executables to ensure the same versions of the dependencies and avoid build failures.